contracts/infrastructure/PlatformVoter.sol

```diff
@@ -113,7 +113,7 @@ contract PlatformVoter is ControllableV3, IPlatformVoter {
     Vote[] memory _votes = votes[tokenId];
```

Comment:
There are no restrictions to call poke. Probably it worth to make a comment why the restrictions are not required.